packer_templates/debian/debian-11.3-amd64.json

```diff
@@ -208,16 +208,16 @@
     "http_directory": "{{template_dir}}/http",
     "http_proxy": "{{env `http_proxy`}}",
     "https_proxy": "{{env `https_proxy`}}",
-    "iso_checksum": "45c9feabba213bdc6d72e7469de71ea5aeff73faea6bfb109ab5bad37c3b43bd",
-    "iso_name": "debian-11.2.0-amd64-netinst.iso",
+    "iso_checksum": "7892981e1da216e79fb3a1536ce5ebab157afdd20048fe458f2ae34fbc26c19b",
```

Comment:
@mmguero can you please let me know from where did you get this SHA value from.

For amd64 SHA. This link I [found](https://cdimage.debian.org/debian-cd/current/amd64/iso-cd/SHA512SUMS) and value didn't match

Reply:
i believe these are SHA256 checksums, appear to match - https://cdimage.debian.org/debian-cd/current/amd64/iso-cd/SHA256SUMS.
@sanjain-progress the link you shared is for SHA512 checksums that are longer.